karton/classifier/classifier.py

```diff
@@ -24,6 +27,21 @@ def classify_openxml(content: bytes) -> Optional[str]:
     return None
 
 
+def load_yara_rules(path: Path) -> yara.Rules:
+    rule_files = {}
+    for f in path.glob("*.yar"):
```

Comment:
Maybe we could consider recursively adding yar files? In the meantime, I've added this as a caveat. Also `.yar` extension is a caveat, I know some people use `.yara` - suggested it as another caveat.